rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -768,3 +774,59 @@ def test_hop_balances_staking(
             amount=reward_amount, usd_value=reward_amount * FVal(1.5),
         ),
     }
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x0e414c1c4780df6c09c2f1070990768D44B70b1D']])
+def test_gearbox_balances(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_transaction_decoder: 'EthereumTransactionDecoder',
+        ethereum_accounts: list[ChecksumEvmAddress],
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
```

Comment:
question, the inquirer is required in this test?

Reply:
Yes, otherwise, it raises this 
```python
AttributeError: type object 'Inquirer' has no attribute '_cached_current_price'
```